### PR TITLE
Added meta tags, title, favicon

### PR DESCRIPTION
**Pull request by duongnguyen0410**
Changes:
- Updated the title tags
- Added images for favicon and meta banner
- Added meta tags, including OG and Twitter tags
- Added favicon link tag

Test cases checked:
- Verified the title showing up

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,7 +2,20 @@
...
+    <meta name="keywords" content="data science, software development, jobs">
+    <meta name="author" content="Duong">
+    <meta name="robots" content="index, follow">
+    <meta property="og:title" content="Opening Jobs">
+    <meta property="og:description" content="Learn about job openings at Duong and submit your application now">
+    <meta property="og:image" content="./meta.png">
+    <meta name="twitter:title" content="Opening Jobs">
+    <meta name="twitter:description" content="Learn about job openings at Duong and submit your application now">
+    <meta name="twitter:image" content="./meta.png">
+    <title>Opening Jobs</title>
```
Comment (ghost) [inline src/index.html]: Make the title longer please, maybe at least 24 characters.

File: src/index.html [hunk 2]
```diff
@@ -2,7 +2,20 @@
 <html>
 
 <head>
-    <title>Ferdi Sahin</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Learn about job openings at Duong and submit your application now">
```
Comment (ghost) [inline src/index.html:7]: This can be improved a bit.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (duongnguyen0410) [reply src/index.html:7]: I think this is fine as it is

File: src/index.html (hunk 1 above)
Comment (duongnguyen0410) [reply src/index.html]: I think it is fine as it is.

File: src/index.html (hunk 2 above)
Comment (duongnguyen410) [reply src/index.html:7]: I think it is fine as it is.

Comment (duongnguyen0410) [reply src/index.html:7]: I think it is fine as it is.

**Review by ghost — APPROVED** [review]

**Pull request merged**